### DISCUSSION TITLE
Personalização do Terminal com erro e lenta.

### DISCUSSION THREAD

**Discussion by dudinhabranco** [balta-io/forum#37] [Tira Dúvidas] [answered]
Tentei seguir as orientações do blog para personalizar o Windows Terminal mas ficou terrivelmente feio e lento para carregar. Tenho a impressão de mudaram os requisitos além de ficar mostrando diversos avisos de que houve uma mudanças significativas na forma de instalar entre outros detalhes.
Visualmente ficou ruim, como se apesar de instalar as fontes novas e selecioná-la nas configurações do WT, não utilizava a fonte correta.
E a outra coisa que ficou pior mesmo foi a demora para abrir o WT que passou a demorar de 10 a 24 segundos.
Acredito que o post do blog, que ensina com personalisar o WTY com os recursos que o Balta usa, precisa ser atualizado.
Alguém conseguiu realizar a instalação e configuração recentemente?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @dudinhabranco, como vai?

Este vídeo pode te ajudar:
https://www.youtube.com/watch?v=1o3wWVG6cnE

Em adicional, este aqui também:
https://www.youtube.com/watch?v=vimeN6uu8jU

**Comment by johnbh3** [comment]
oh-my-posh : O termo 'oh-my-posh' não é reconhecido como nome de cmdlet, função, arquivo de script ou programa
operável. Verifique a grafia do nome ou, se um caminho tiver sido incluído, veja se o caminho está correto e tente
novamente.
No C:\Users\johnbh3\Documents\WindowsPowerShell\Modules\oh-my-posh\7.85.2\oh-my-posh.psm1:89 caractere:5
+     oh-my-posh init pwsh --config="$config" | Invoke-Expression
+     ~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (oh-my-posh:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Reply by andrebaltieri** [reply]
Bom dia, @johnbh3 pelo erro o Windows não conseguiu adicionar o Oh-my-posh nas variáveis de ambiente...
Tenta fazer a instalação via `winget` como mostrado aqui:
https://ohmyposh.dev/docs/installation/windows
